content/en/tracing/troubleshooting/_index.md

```diff
@@ -164,12 +164,14 @@ var tracer = Tracer.Create(isDebugEnabled: true);
 Tracer.Instance = tracer;
 ```
 
-Location of the profiler log:
+The environment variable `DD_TRACE_DEBUG` can also be set to `true`.
+
+Logs (Profiler and Tracer) are saved in the following directories:
```

Comment:
The difference between the Profiler and the Tracer are internal details that most users are not familiar with. As far as they know, they install a single product, the ".NET Tracer".
```suggestion
Logs files are saved in the following directories:
```